@here/harp-mapview/lib/text/TextElementsRendererOptions.ts

```diff
@@ -34,6 +34,13 @@ const DEFAULT_LABEL_DISTANCE_SCALE_MIN = 0.7;
  */
 const DEFAULT_LABEL_DISTANCE_SCALE_MAX = 1.5;
 
+// Allowed distance to screen border for early rejection of POIs during placement. Its range is
+// [0..1] of screen size.
+// A value of 0 will lead to POI labels popping in at the border of the screen. A large value will
+// lead to many labels being placed outside the screen, with all the required actions for measuring
+// and loading glyphs impacting performance.
+const DEFAULT_MAX_DISTANCE_TO_BORDER = 0.2;
```

Comment:
Check the performance with 0.1, that might make a difference.

Reply:
I did test manually, and at 0.1 there are a few larger labels that are negatively affected, i.e. they disappear too soon during map panning.